crowbar_framework/app/models/node_object.rb

```diff
@@ -1072,11 +1072,22 @@ def file_exist?(file)
     out[:exit_code].zero?
   end
 
+  def script_finished(script)
+    base = "/var/lib/crowbar/upgrade/" + File.basename(script, ".sh")
+    ok_file = base + "-ok"
+    failed_file = base + "-failed"
+    return "ok" if file_exist? ok_file
```

Comment:
can we use symbols here for the `return` value as in the upgrade status? that would minimally enhance the memory footprint